### PR TITLE
Add aspire example

### PR DESCRIPTION
**Pull request by liammclennan**
Basic setup of Aspire with Seq

### CODE REVIEW

File: client/aspire-with-seq/AspireWithSeq.AppHost/AspireWithSeq.AppHost.csproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <IsAspireHost>true</IsAspireHost>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\AspireWithSeq.ApiService\AspireWithSeq.ApiService.csproj" />
+    <ProjectReference Include="..\AspireWithSeq.Web\AspireWithSeq.Web.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Aspire.Hosting" Version="8.0.0-preview.4.24105.20" />
```
Comment (nblumhardt) [inline client/aspire-with-seq/AspireWithSeq.AppHost/AspireWithSeq.AppHost.csproj:17]: These are GA now, looks like 8.0.1 is the current version?

File: client/aspire-with-seq/AspireWithSeq.AppHost/Program.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var apiService = builder.AddProject<Projects.AspireWithSeq_ApiService>("apiservice");
+
+builder
+    .AddResource(new ContainerResource("seq"))
+    .WithAnnotation(new EndpointAnnotation(ProtocolType.Tcp, uriScheme: "http", name: "seq", port: 5341, containerPort: 80))
+    .WithEnvironment("ACCEPT_EULA", "Y")
+    .WithAnnotation(new ContainerImageAnnotation { Image = "datalust/seq", Tag = "latest" });
```
Comment (nblumhardt) [inline client/aspire-with-seq/AspireWithSeq.AppHost/Program.cs:11]: Does this overlap with what `Aspire.Hosting.Seq` provides?

**Review by nblumhardt — COMMENTED** [review]
Just a query about where the official Aspire Seq packages fit in (I'm not entirely across the state of these, sorry :-))

**Comment by liammclennan** [comment]
Going to close this one. We can add a GA version later if it's worthwhile.

**Pull request closed without merging**